sql_generators/glean_usage/templates/event_monitoring_aggregates_v1.query.sql

```diff
@@ -82,9 +82,7 @@
               ) AS extra
             ) AS value
           FROM
-            UNNEST(ping_info.experiments) WITH OFFSET
-          ORDER BY
-            offset
+            UNNEST(ping_info.experiments)
```

Comment:
I'm guessing you're removing this for performance reasons, and because the order doesn't really matter?

Reply:
Yes I don't expect any noticeable difference but I'm curious to see